### PR TITLE
Add transaction limit amount param definitions

### PR DESCRIPTION
**Pull request by nanobash**
- Add transaction limit amount param definitions

### CODE REVIEW

File: openapi/components/schemas/Transactions/Transaction.yaml [hunk 1]
```diff
@@ -253,6 +253,9 @@ allOf:
         readOnly: true
         nullable: true
         x-sortable: true
+      limits:
+        allOf:
```
Comment (adamaltman) [inline openapi/components/schemas/Transactions/Transaction.yaml:257]: allOf isn't required here.

File: openapi/components/schemas/Transactions/TransactionRequest.yaml [hunk 2]
```diff
@@ -24,3 +24,6 @@ allOf:
           - sale
           - authorize
           - setup
+      limits:
+        allOf:
```
Comment (adamaltman) [inline openapi/components/schemas/Transactions/TransactionRequest.yaml:28]: allOf isn't required here.

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/Transactions/Transaction.yaml (hunk 1 above)
Comment (nanobash) [reply openapi/components/schemas/Transactions/Transaction.yaml]: Removed, thank you

File: openapi/components/schemas/Transactions/TransactionRequest.yaml (hunk 2 above)
Comment (nanobash) [reply openapi/components/schemas/Transactions/TransactionRequest.yaml]: Removed, thank you

**Review by adamaltman — DISMISSED** [review]

**Review by justnero — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

**Review by vykhrystiuk — DISMISSED** [review]

File: openapi/components/schemas/TransactionLimitAmount.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    type: number
+    format: double
+    example: 275.35
+    description: The limit amount.
+  currency:
+    $ref: ./CurrencyCode.yaml
+  resetTime:
+    type: string
+    format: date-time
+    description: The time the limit resets which may be used for some user interfaces.
```
Comment (noon-dawg) [inline openapi/components/schemas/TransactionLimitAmount.yaml:15]: suggestion: At first read I was not sure which time was being reset.
I understood on the second or third read. My suggestion here is to help make this more explicit.

```suggestion
    description: The date and time in which the limit amount resets. This value may be used for some user interfaces.
```

**Comment by nanobash** [comment]
@Rebilly/php please review

**Review by ashkarpetin — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Review by justnero — APPROVED** [review]

**Pull request merged**